spot_utils/perception/conversion.py

```diff
@@ -0,0 +1,52 @@
+"""Functions for converting between data types."""
+
+from typing import List
+
+import numpy as np
+from bosdyn.client.image import _depth_image_get_valid_indices
+
+from spot_utils.structures.image import PointCloud, RGBDImage
+
+
+def rgbd_to_pointcloud(
```

Comment:
could we just use this instead?

http://www.open3d.org/docs/0.6.0/python_api/open3d.geometry.create_point_cloud_from_rgbd_image.html

Reply:
This is a modified version of the one from the BD library: https://github.com/boston-dynamics/spot-sdk/blob/master/python/bosdyn-client/src/bosdyn/client/image.py#L396

So I know for sure it meshes with the output image responses. Also see above.